Remotion/Web/Development.WebTesting.IntegrationTests/BrowserSessionTest.cs

```diff
@@ -70,6 +70,22 @@ public void ChromeDriver_SupportsBrowserLogs ()
       Assert.That (logs.Count (l => l.Message.Contains (warningMessage)), Is.EqualTo (1));
     }
 
+    [Test]
+    public void GeckoDriver_DoesNotSupportBrowserLogs ()
+    {
+      if (!Helper.BrowserConfiguration.IsFirefox())

```

Comment:
Okay, right now this only affects the trunk/devlop. But we will need some logic here to deal with browser logs in firefox.

Reply:
Okay, seen production code. please change to use two paths in the test, one for firefox, one for the rest and asser tthe getlog() result in both cases.